src/main/java/com/internet/shop/controllers/orders/ShowCurrentUserOrdersController.java

```diff
@@ -0,0 +1,27 @@
+package com.internet.shop.controllers.orders;
+
+import com.internet.shop.lib.Injector;
+import com.internet.shop.model.Order;
+import com.internet.shop.service.OrderService;
+import java.io.IOException;
+import java.util.List;
+import javax.servlet.ServletException;
+import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+
+public class ShowCurrentUserOrdersController extends HttpServlet {
+    private static final Long USER_ID = 1L;
+
+    private static final Injector injector = Injector.getInstance("com.internet.shop");
+    private final OrderService orderService =
+            (OrderService) injector.getInstance(OrderService.class);
+
+    @Override
+    protected void doGet(HttpServletRequest req, HttpServletResponse resp)
+            throws ServletException, IOException {
+        List<Order> orders = orderService.getUserOrders(USER_ID);
+        req.setAttribute("orders", orders);
+        req.getRequestDispatcher("/WEB-INF/views/orders/currentUserOrders.jsp").forward(req, resp);
```

Comment:
I might be wrong but in the naming of folders ( .../orders/currentUsersOrders) your repeat word "orders" - might be not critical and very difficult to suggest something meaningful. May be - "/orders/byUser"?